src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -207,6 +207,29 @@ public function getTypesFromConstructor(string $class, string $property): ?array
 
     public function getType(string $class, string $property, array $context = []): ?Type
     {
+        // BC layer, remove context key in 8.0
+        $extractPublicPropertiesFirst = \array_key_exists('extract_public_properties_first', $context);
+        if (!$extractPublicPropertiesFirst) {
+            trigger_deprecation('symfony/property-info', '7.3', 'Not setting "extract_public_properties_first" context key is deprecated, it will default to "true" in 8.0.');
```

Comment:
```suggestion
        $extractPublicPropertiesFirst = $context['extract_public_properties_first'] ?? false;

        if (!$extractPublicPropertiesFirst) {
            trigger_deprecation('symfony/property-info', '7.3', 'Not setting "extract_public_properties_first" context value to "true" is deprecated, it will always be considered as "true" in 8.0.');
```